cartridges/LINK_adyen/cartridge/templates/default/checkout/billing/paymentOptions.isml

```diff
@@ -2,6 +2,8 @@
     var assets = require('*/cartridge/scripts/assets.js');
     assets.addJs('/js/adyen-cse.js');
     assets.addCss('/css/adyenCss.css');
+    assets.addJs('/js/adyenSecureFields.js');
```

Comment:
if you use the card components I think this js is not necessary anymore.
Maybe the same is true for the css under this? (I'm not sure)